frontend/simple/views/Invite.vue

```diff
@@ -21,31 +88,50 @@
                 </a>
               </p>
           </div>
+          <i18n v-if="error" id="badUsername" class="help is-danger">Invalid Username</i18n>
+          <i18n v-if="self" class="help is-danger">Cannot Invite Yourself</i18n>
         </form>
       </div>
     </div>
 
     <div class="columns">
       <div class="column is-6 is-offset-3" >
-
-        <p
-          class="notification is-success has-text-centered"
-          v-if="invited"
+        <table
+                class="table is-bordered is-striped is-narrow"
+                v-if="candidateMembers.length"
         >
-          <i class='notification-icon fa fa-check'></i>
-          <i18n>Members invited successfully!</i18n>
-        </p>
-
-        <i18n v-if="error" id="badUsername" class="help is-danger">Invalid Username</i18n>
-        <i18n v-if="self" class="help is-danger">Cannot Invite Yourself</i18n>
-
+          <thead>
+          <tr>
+            <th class="table-header"><i18n>Candidate Members</i18n></th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr v-for="(candidate, index) in candidateMembers" class="vote">
+            <td>
+              <div class="media">
+                <div class="media-left">
+                  <p class="image is-64x64">
+                    <!-- TODO: use responsive figure:
+                  http://bulma.io/documentation/elements/image/ -->
+                    <!-- TODO: ideally these would be loaded from cache -->
+                    <img src="http://bulma.io/images/placeholders/128x128.png">
```

Comment:
Remember: it's best not to load placeholders from external websites, and we now have one locally I believe